src/reducers/books.js

```diff
@@ -0,0 +1,21 @@
+const CREATE_BOOK = 'CREATE_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const initState = [
+  { title: 'Harry Potter', category: 'Kids', id: Math.random() },
+  { title: 'Steve Jobs', category: 'Biography', id: Math.random() },
+  { title: 'Microverse', category: 'Learning', id: Math.random() },
+];
+
```

Comment:
requirements prescribes this code instead to be in the file `"src/index.js"` 
[ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) ]

Reply:
We will do this my bad